modules/sandboxed-containers-deleting-namespace-cli.adoc

```diff
@@ -0,0 +1,25 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-deleting-namespace-cli_{context}"]
+= Deleting {sandboxed-containers-operator} namespace
+
+After you run the preceding commands, your cluster is restored to the state that it was prior to the installation process. You can now revoke namespace access to the operator by deleting the `openshift-sandboxed-containers-operator` namespace.
+
+.Prerequisites
+
+* You have {product-title} 4.10 installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the comand-line JSON processor (`jq`).
```

Comment:
```suggestion
* You have installed the command-line JSON processor (`jq`).
```

Reply:
Done. This file was superfluous and has been deleted.

Reply:
This section was deleted.